app/views/ansible_roles/index.html.erb

```diff
@@ -1,45 +1,31 @@
+<%= webpacked_plugins_js_for :foreman_ansible %>
+<%= webpacked_plugins_css_for :foreman_ansible %>
 <% title _("Ansible Roles") %>
 
 <% title_actions ansible_proxy_import(hash_for_import_ansible_roles_path),
   documentation_button('#4.1ImportingRoles', :root_url => ansible_doc_url) %>
 
-<table class="<%= table_css_classes 'table-fixed' %>">
-  <thead>
-    <tr>
-      <th class="col-md-6"><%= sort :name, :as => s_("Role|Name") %></th>
-      <th class="col-md-2"><%= _("Hostgroups") %></th>
-      <th class="col-md-2"><%= _("Hosts") %></th>
-      <th class="col-md-2"><%= sort :updated_at, :as => _("Imported at") %></th>
-      <th class="col-md-2"><%= _("Actions") %></th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @ansible_roles.each do |role| %>
-      <tr>
-        <td class="ellipsis"><%= role.name %></td>
-        <td class="ellipsis"><%= role.hostgroups.count %></td>
-        <td class="ellipsis"><%= link_to role.hosts.count, hosts_path(:search => "ansible_role = #{role.name}")%></td>
-        <td class="ellipsis"><%= import_time role %></td>
-        <td>
-          <%
-              links = [
-                link_to(
-                  _('Variables'),
-                  ansible_variables_path(:search => "ansible_role = #{role}")
-                ),
-                display_delete_if_authorized(
-                  hash_for_ansible_role_path(:id => role).
-                  merge(:auth_object => role, :authorizer => authorizer),
-                  :data => { :confirm => _("Delete %s?") % role.name },
-                  :action => :delete
-                )
-              ]
-          %>
-          <%= action_buttons(*links) %>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
 
-<%= will_paginate_with_info @ansible_roles %>
+
+<% rows = [] %>
+<%   @ansible_roles.each do |role| %>
```

Comment:
Could we prepare rows in a helper method?

Reply:
sure